jsonschema/tests/test_cli.py

```diff
@@ -173,3 +226,111 @@ def test_piping(self):
         self.assertFalse(stdout.getvalue())
         self.assertFalse(stderr.getvalue())
         self.assertEqual(exit_code, 0)
+
+    def test_schema_parsing_error(self):
```

Comment:
Perhaps a test for providing an unknown output format would be nice. The way things are implemented (via `choices`) you probably just can call `parse_args`, since the actual function won't run.

Reply:
I added a test in `TestParser`, is that what you were thinking about ?

Reply:
Not sure I see it, maybe I'm missing it?

I'm talking about what happens if someone passes `--output=gibberish`.

Reply:
It is done by `test_unknown_output`, I’m writing about it [in this comment](https://github.com/Julian/jsonschema/pull/661#discussion_r390155906)

Reply:
Ah I see now, thanks!